src/react/deckgl.js

```diff
@@ -55,7 +56,8 @@ const defaultProps = {
   onWebGLInitialized: noop,
   onAfterRender: noop,
   onLayerClick: null,
-  onLayerHover: null
+  onLayerHover: null,
+  useDevicePixelRatio: false
```

Comment:
@ibgreen ,  this will change existing apps behavior to use non-retina resolution, do you think I should use `true` for default, then set to false only for mobile devices?